dev/dags/dbt/jaffle_shop/models/orders.sql

```diff
@@ -1,3 +1,9 @@
+{{ config(
+    materialized='view',
+    alias=var('animation_alias', 'orders')
```

Comment:
Why are we naming an alias to `orders` as `animation_alias`?

Reply:
Makes sense, shall we rename `animation_alias` to `orders_alias`?

Reply:
Fix in: https://github.com/astronomer/astronomer-cosmos/pull/1366/commits/8fd03dc8afda10a51b3185c5c5f744b4bf9fe28d